### PR TITLE
Makefile,build/release-materials: initial release generation tooling.

### PR DESCRIPTION
**Pull request by erikh**
This doesn't need to pass tests, just for releasing new versions.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -120,9 +120,9 @@ run-apiserver:
 
 run-build:
 	GOGC=1000 go install -v \
-		-ldflags '-X main.version=$(if $(BUILD_VERSION),$(BUILD_VERSION),devbuild)' \
-		./volcli/volcli/ ./volplugin/volplugin/ ./apiserver/apiserver/ ./volsupervisor/volsupervisor/ ./volmigrate/volmigrate/
-	cp $(GUESTBINPATH)/* bin/
+		 -ldflags '-X main.version=$(if ${BUILD_VERSION},${BUILD_VERSION},devbuild)' \
```
Comment (dseevr) [inline Makefile]: Is your editor adding the extra spaces at the beginning of the lines here?  I removed them in the commit which added volmigrate

Comment (erikh) [inline Makefile]: could be a formatting issue

Comment (erikh) [inline Makefile]: oh right. I fixed this as a result of a merge confict I had after we merged your branch. I will resolve this in a minute.

**Comment by yuva29** [comment]
@erikh Sorry I'm not reviewing this atm. Need to explore Ruby.

**Comment by erikh** [comment]
all good.

**Comment by erikh** [comment]
this is all shell though :D

**Comment by erikh** [comment]
@dseevr resolved the whitespace issues, thanks for noticing

**Comment by dseevr** [comment]
CI failed:

```

**Comment by dseevr** [comment]
nvm that comment, I came back to this from another tab and forgot the comment on the top.

**Pull request merged**